onnxruntime/test/providers/vitisai/vitisai_basic_test.cc

```diff
@@ -0,0 +1,316 @@
+#pragma comment(lib, "d3d12.lib")
+
+#include "core/providers/vitisai/imp/tensor_proto.h"
```

Comment:
Please configure CMake to find that header automatically.

Reply:
I added path to header file using "core/providers/..." according to other execution providers [test source codes](https://github.com/apwojcik/onnxruntime/tree/uif2/develop/onnxruntime/test/providers). For example, I was looking at CUDA execution providers test code - [here](https://github.com/apwojcik/onnxruntime/blob/uif2/develop/onnxruntime/test/providers/cuda/test_cases/cuda_test_provider.cc) is the example.

If you think that it should be changed inside CMake, I can do that too.